src/RoboFile/ClientContainer.php

```diff
@@ -42,7 +42,24 @@ public function runContainerTests(
 		$opts['install-test'] = '';
 
 		$this->runContainerTestPreparation($opts);
+
 		$this->runTestSuites($opts);
+
+	}
+
+	public function runContainerTest(
```

Comment:
Well, it actually does something specific to the project. It changes the acceptance yml file according to the specified server.